src/main/daml/Daml/Finance/Interface/Types/Common/Types.daml

```diff
@@ -21,6 +21,29 @@ newtype Id = Id Text
 instance Show Id where
   show (Id t) = t
 
+-- | The `HoldingStandard` data type classifies holdings based on the interfaces they implement in
+-- the holding hierarchy:
+--
+--   Fungible.I
+--      | require
+--      v
+--   Transferable.I
+--      | require
+--      v
+--    Base.I
+--
+data HoldingStandard
```

Comment:
I wonder if it might be worth renaming these to
- Base
- Transferable
- Fungible

Wdyt? Can discuss it with the team. 

Also we should explain why we believe that a `Fungible` is usually `Transferable`

Reply:
I agree, I will rename them

Reply:
I will rename in a separate PR

Reply:
@matteolimberto-da Maybe we can consider it in a subsequent PR, I added a commit for ^ in this PR https://github.com/digital-asset/daml-finance/pull/1064 (which builds on top of this one)

Reply:
agree, latter 2 points are for the next PR
https://github.com/digital-asset/daml-finance/pull/1064